core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -44,7 +44,7 @@ import org.apache.spark.broadcast.Broadcast
 import org.apache.spark.deploy.SparkHadoopUtil
 import org.apache.spark.executor.DataReadMethod
 import org.apache.spark.rdd.HadoopRDD.HadoopMapPartitionsWithSplitRDD
-import org.apache.spark.util.{SerializableConfiguration, NextIterator, Utils}
+import org.apache.spark.util.{SerializableConfiguration, NextIterator, Utils, ShutdownHookManager}
```

Comment:
Indeed, they didn't look like they were sorted in the first place..  So I tried to leave it that way everywhere.  But I agree with you that it's better sorted.